stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -596,7 +600,9 @@ extension Tensor where Scalar : TensorFlowFloatingPoint {
     let value = mean(squeezingAxes: axes)
     return (value, { [shape = shapeTensor,
                       count = axes.map { shape[$0] }.reduce(1, *)] in
-      $0.broadcast(toShape: shape) / Tensor(Scalar(count))
+      var res = $0
```

Comment:
As you noted, this function appears to never be called. Could you please delete it?

Reply:
Never mind; I have done so in https://github.com/apple/swift/pull/24164/commits/fb565e0dffb880c4aedf9d2ba6a116b458bcc88c.